pkg/route/api/allocator/allocator.go

```diff
@@ -0,0 +1,30 @@
+package allocator
```

Comment:
Essentially the interface should only expose the input (route) and the output (key to router), and then the interface would be used by the allocator (which won't be in the rest call).  In fact, allocations like this being in the rest call are deceptive and should have a clear TODO: this does not belong here


> On Mar 10, 2015, at 1:42 PM, ramr <notifications@github.com> wrote:
> 
> In pkg/route/api/allocator/allocator.go:
> 
> > @@ -0,0 +1,30 @@
> > +package allocator
> Ok - will do. Though I suspect this may need some change when the sharding support is added.
> 
> —
> Reply to this email directly or view it on GitHub.
>